buildSrc/src/main/java/org/springframework/boot/build/mavenplugin/PluginXmlParser.java

```diff
@@ -32,6 +32,8 @@
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 
+import static java.util.Objects.requireNonNull;
```

Comment:
We don't use static import in production code.

Reply:
Removed as of commit 976fc62